src/pages/workspace/companyCards/WorkspaceCompanyCardsFeedAddedEmptyPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import EmptyStateComponent from '@components/EmptyStateComponent';
+import * as Illustrations from '@components/Icon/Illustrations';
+import TableListItemSkeleton from '@components/Skeletons/TableRowSkeleton';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+function WorkspaceCompanyCardsFeedAddedEmptyPage() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    return (
+        <EmptyStateComponent
+            SkeletonComponent={TableListItemSkeleton}
+            headerMediaType={CONST.EMPTY_STATE_MEDIA.ILLUSTRATION}
+            headerMedia={Illustrations.CompanyCardsEmptyState}
+            headerStyles={styles.emptyCompanyCardsFolderBG}
```

Comment:
```suggestion
            headerStyles={[styles.emptyStateCardIllustrationContainer, styles.justifyContentStart, {backgroundColor: colors.blue700}]}
```
Height of this needs to be 220px, so we can use the same styles which used in `WorkspaceCompanyCardPageEmptyState`. With this, we can take out extra styles and theme changes.

Reply:
Updated